### PR TITLE
boards: merge repo. boards as folder

### PR DESCRIPTION
**Pull request by luisan00**
### Contribution description
First step in the groupping repositories task

### Testing procedure
make should work with m4a boards:

```sh
make BOARD=m4a-mb
```
or
```sh
make BOARD=m4a-24g
```
should build the firmware

### Issues/PRs references
#69

### CODE REVIEW

**Review by eduazocar — COMMENTED** [review]
Everything it's fine :+1:

**Review by eduazocar — APPROVED** [review]

**Comment by eduazocar** [comment]
I see some warnings, from a group of files in example folder...

**Comment by eduazocar** [comment]
squash to merge soon :+1:

**Comment by luisan00** [comment]
> squash to merge soon fireworks

Today should be ready to merge

**Review by eduazocar — CHANGES_REQUESTED** [review]

File: boards/m4a-mb/include/board.h [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+ * @{
+ */
+#define LED_PORT PORT->Group[PB]                        /*!< Group is Port B */
+/** @} */
+
+/**
+ * @name   LED D0 (orange) definition and handler
+ * @{
+ */
+#define LED0_PIN GPIO_PIN(PB, 30)                       /*!< LED0 is routed to pin PB30 in the MCU */
```
[the remark this answers is not in the document]
Comment (luisan00) [reply boards/m4a-mb/include/board.h:49]: :+1:

**Pull request merged**